sanic/touchup/schemes/base.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+from typing import Set, Type
+
+
+class BaseScheme(ABC):
+    ident: str
+    _registry: Set[Type] = set()
+
+    def __init__(self, app) -> None:
+        self.app = app
+
+    @abstractmethod
+    def run(self, method, module_globals) -> None:
+        ...
+
+    def __init_subclass__(cls):
+        BaseScheme._registry.add(cls)
+
+    @property
+    def trigger(self):
+        return f"# {self.ident}: "
```

Comment:
ehhh ... what's the purpose of this method?

Reply:
:thinking: I'm not sure. Did I add that? :laughing: 

It looks to me like it might be a relic of development and can be deleted. I will confirm.

Reply:
well, I leave breadcrumbs everywhere in my code (specially development stuff), so ... :shrug: